recipes/hdf5/build.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+if [ `uname -m` == ppc64le ]; then
+    B="--build=ppc64le-linux"
+fi
+
+./configure $B --prefix=$PREFIX \
+    --enable-linux-lfs --with-zlib \
+    --with-pthread=yes  --enable-cxx --with-default-plugindir=$PREFIX/lib/hdf5/plugin
```

Comment:
Sure. Not saying it needs to change.

Though I do appreciate that you have tried this and found this does not to work. Good thing to remember in the future. Happy to stick with the standard configure.